templates/firefox-ios.xliff

```diff
@@ -2761,16 +2786,20 @@
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
     </header>
     <body>
+      <trans-unit id="TodayWidget.CopiedLinkLabelFromPasteBoard">
+        <source>Copied Link from clipboard</source>
+        <note>Copied Link from clipboard displayed</note>
+      </trans-unit>
       <trans-unit id="TodayWidget.GoToCopiedLinkLabel">
         <source>Go to copied link</source>
         <note>Go to link on clipboard</note>
       </trans-unit>
       <trans-unit id="TodayWidget.NewPrivateTabButtonLabel">
-        <source>New Private Tab</source>
+        <source>Private Search</source>
         <note>New Private Tab button label</note>
       </trans-unit>
       <trans-unit id="TodayWidget.NewTabButtonLabel">
```

Comment:
This is more of a refactoring. We already had these strings and moved it to this file. 
Since this is an older string would it be okay if I leave the same comment @Delphine ?